strophejingle/index.html

```diff
@@ -95,15 +117,18 @@
                 setTimeout(function() { waitForRemoteVideo(selector, sid); }, 100);
             }
         }
+
         var sess = connection.jingle.sessions[sid];
         var vid = document.createElement('video');
-        var id = 'remoteVideo_' + sid + '_' + data.stream.id;
+        var id = 'remoteVideo_' + sid + '_default';
```

Comment:
works for me in both chrome and firefox and the MediaStream is supposed to have an id